de.lernsoftware_filius.Filius.json

```diff
@@ -0,0 +1,85 @@
+{
+  "app-id": "de.lernsoftware_filius.Filius",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "21.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.openjdk11"],
+  "command": "de.lernsoftware_filius.Filius",
+  "finish-args": [
+    "--env=PATH=/app/jre/bin:/usr/bin:/app/bin",
+    "--share=network",
+    "--socket=x11",
+    "--share=ipc",
+    "--device=dri",
+    "--socket=pulseaudio",
+    "--filesystem=host"
+  ],
+  "modules": [
+    {
+      "name": "openjdk",
+      "buildsystem": "simple",
+      "build-commands": ["/usr/lib/sdk/openjdk11/install.sh"]
+    },
+    {
+      "name": "filius",
+      "buildsystem": "simple",
+      "build-commands": [
+        "mkdir -p /app/filius",
+        "cp -r * /app/filius",
+        "chmod +x /app/filius/filius.sh"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://www.lernsoftware-filius.de/downloads/Setup/filius-1.12.5.zip",
+          "sha256": "3334ec05947d31c864b72802d6af623628bb42a8dca60ced3c042fb27475053a",
+          "strip-components": 0
+        }
+      ]
+    },
+    {
+      "name": "filius-icon",
+      "buildsystem": "simple",
+      "build-commands": [
+        "for size in 16 32; do install -Dm644 filius${size}.png /app/share/icons/hicolor/${size}x${size}/apps/de.lernsoftware_filius.Filius.png; done"
```

Comment:
you are missing a 128x128 icon.

also the 64x64 icon isn't installed here.

Reply:
done